community-operators/service-binding-operator/0.0.20/service-binding-operator.v0.0.20.clusterserviceversion.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       kind: ServiceBindingRequest
       name: servicebindingrequests.apps.openshift.io
       version: v1alpha1
-  description: Placeholder description
+  description: 'The Service Binding Operator enables users to create a managed binding between applications and operator-backed services such as databases, without having to perform manual configuration of secrets, configmaps etc.'
```

Comment:
@DhritiShikhar , could we remove the single quotes `'`....`'`

Reply:
Done